lib/HTTP/Status/unofficial.pm

```diff
@@ -46,6 +46,4 @@ $StatusCode{'509'} = 'Bandwidth Limit Exceeded';
 
 =cut
 
-use DDP; p %StatusCode;
```

Comment:
this commit should have been squashed with the original that added this line, so we never saw it. Splitting it out into a separate change adds no value.

Reply:
:-( sorry, it was already done